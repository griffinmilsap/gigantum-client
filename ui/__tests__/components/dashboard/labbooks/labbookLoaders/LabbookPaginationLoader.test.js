import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import LabbookPaginationLoader from 'Components/dashboard/shared/loaders/CardLoader';
import relayTestingUtils from '@gigantum/relay-testing-utils';

describe('Snapshot LabbookPaginationLoader', () => {
  it('LabbookPaginationLoader isLoading true', () => {
    const datasets = renderer.create(

       <LabbookPaginationLoader isLoadingMore={true}/>,

    );

    const tree = datasets.toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('LabbookPaginationLoader isLoading false', () => {
    const datasets = renderer.create(

       <LabbookPaginationLoader isLoadingMore={false}/>,

    );

    const tree = datasets.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
