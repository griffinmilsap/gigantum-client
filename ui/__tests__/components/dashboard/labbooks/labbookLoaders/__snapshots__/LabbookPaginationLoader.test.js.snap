// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot LabbookPaginationLoader LabbookPaginationLoader isLoading false 1`] = `
<div
  className="CardLoader--hidden"
  data-selenium-id="CardLoader"
>
  <div />
</div>
`;

exports[`Snapshot LabbookPaginationLoader LabbookPaginationLoader isLoading true 1`] = `
<div
  className="Card Card--300 column-4-span-3 flex flex--column justify--space-between CardLoader CardLoader--undefined"
  data-selenium-id="CardLoader"
>
  <div />
</div>
`;
